### PR TITLE
Add storage_encrypted argument

### PR DESCRIPTION
**Pull request by drmikecrowe**
## What
* Add storage_encrypted argument
## Why
* We need to have option for RDS storage encryption

### CODE REVIEW

**Comment by osterman** [comment]
Thanks! Will have someone test this out this week and get it merged ASAP.

File: variables.tf [hunk 1]
```diff
@@ -38,6 +38,11 @@ variable "storage_type" {
   default     = "standard"
 }
 
+variable "storage_encrypted" {
+  description = "(Optional) Specifies whether the DB instance is encrypted. The default is false if not specified."  
+  default     = false
```
Comment (aknysh) [inline variables.tf]: Use strings "false" and "true" for boolean values, _e.g._ `default     = "false"`.
Terraform does not work with plain booleans correctly in many cases, and they recommend using strings.

File: tests/rds_instance.tf [hunk 2]
```diff
@@ -13,6 +13,7 @@ module "rds_instance" {
   multi_az                    = true
   storage_type                = "gp2"
   allocated_storage           = "100"
+  storage_encrypted           = false
```
Comment (aknysh) [inline tests/rds_instance.tf]: Use strings "false" and "true" for boolean values, _e.g._ `storage_encrypted  = "false"`.
Terraform does not work with plain booleans correctly in many cases, and they recommend using strings.

File: README.md [hunk 3]
```diff
@@ -77,6 +78,7 @@ module "rds_instance" {
       multi_az                    = true
       storage_type                = "gp2"
       allocated_storage           = "100"
+      storage_encrypted           = true
```
Comment (aknysh) [inline README.md]: storage_encrypted = "true"

**Comment by comeanother** [comment]
Tested
screenshot from 2017-10-31 18-33-06
screenshot from 2017-10-31 18-15-37
screenshot from 2017-10-31 18-18-34

**Comment by comeanother** [comment]
@aknysh we're going to merge these changes, because we need them now.
As @drmikecrowe haven't added changes you've requested, they'll be added as a separate PR.

**Pull request merged**